docs/solarsystem/pds/pds.rst

```diff
@@ -0,0 +1,132 @@
+.. doctest-skip-all
```

Comment:
don't skip tests in the documentation, but use the `doctest-remote-data` directive to flag code snippets that are expected to reach the internet